locale/en/blog/release/v4.1.1.md

```diff
@@ -2,7 +2,7 @@
 date: 2015-09-23T02:01:38.545Z
 version: 4.1.1
 category: release
-title: Node v4.1.1 (Stable)
+title: Node v4.1.1 (Current)
```

Comment:
I'm not under the impression that we will be changing the release type of old releases... these posts should likely remain unchanged